htdocs/bootstrap/css/custom-css.css

```diff
@@ -104,7 +104,7 @@ table th{
     left:0;
     top:auto;
     word-wrap: break-word;
-    border-right: 0px none black;
+    border-right: 0 none black;
     border-top-width:3px; /*only relevant for first row*/
```

Comment:
Should there be spaces between the `:` and the `value` here, as per your other refactors?

Reply:
My personal preference is to have a space in-between the colon and the value. I think it helps developers reading the css slightly. I can go through and fix by adding a space or removing.

Reply:
I agree! I think the space adds clarity. If there's no opposition, I think this PR might as well include spaces between all colons and values.